block_model.rb

```diff
@@ -0,0 +1,73 @@
+class BlockModel
+  attr_reader :blocks
+
+  def initialize(blocks)
+    @blocks = blocks
+  end
+
+  def reblock(rx, ry, rz)
```

Comment:
There should be comments, or indications of what the variables rx, ry and rz are supposed to be, as the variables by themselfs are magic numbers given to the function when it is called